subprojects/netstring-c.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = netstring-c-92104d314ad885fd444b70bffd12a2963ef37a2f
+source_url = https://github.com/PeterScott/netstring-c/archive/92104d314ad885fd444b70bffd12a2963ef37a2f.tar.gz
```

Comment:
Hmm, upstream has no release tags? Last code change seems to be 11 years ago, so I guess the commit sha1 is just as fine...

Reply:
Yeah, I also checked that author doesn't use GitHub lately, so it might be challenging to convince them to create a tag and meaningful version.

Reply:
Using a serialisation framework to be used in network traffic that has not been updated since 2015 seems like a massive security hole waiting to happen.

Reply:
It is a very, very simple serialization format and there is really nothing to be updated in it, works just fine since 2015 :sunglasses: 

Source code is 100 lines, half of which are comments.